### DISCUSSION TITLE
Plugin Update

### DISCUSSION THREAD

**Discussion by surendran-t** [sparkle-project/Sparkle#2523] [Q&A]
Hi Folks, I am trying to use sparkle to update plug-ins by itself. Can any one please provide more insight on how to do this ?.

**Comment by zorgiepoo** [comment]
There is some information about plug-ins on the documentation website [here](https://sparkle-project.org/documentation/bundles#sparkle-2) under the Sparkle 2 section. Look over that first.

Does your plug-in share the same process as the host app and other plug-ins? If so, using Sparkle inside that process is not recommended (instead you can use Sparkle in an external process). Does your plug-in inherit a sandboxed environment from the host app or its parent process? If the plug-in is sandboxed, things can get more tricky.

There isn't any information here on what kind of plug-in/environment you have. Sparkle can update bundles but there can be things to consider if you want to use Sparkle _from_ a plug-in.
